ush/machine/wcoss2.yaml

```diff
@@ -34,8 +34,8 @@ task_run_fcst:
   FIXlut: /lfs/h2/emc/lam/noscrub/UFS_SRW_App/develop/fix/fix_lut
 data:
   GSMGFS: compath.py ${envir}/gsmgfs/${gsmgfs_ver}/gsmgfs.${PDY}
-  FV3GFS: compath.py ${envir}/gfs/${gfs_ver}/gfs.${PDY}
+  FV3GFS: compath.py ${envir}/gfs/${gfs_ver}/gfs.${PDY}/${cyc}/atmos
```

Comment:
Why not use ${hh} here instead of ${cyc}? This is a bit misleading because this ${cyc} is not necessarily aligned with the `cyc` set for the RRFS config.

Then, there's no need to rename cyc in the ex-script.

Reply:
yes, I agree. it has been replaced with 'hh'.